### ISSUE TITLE
about the examples

### ISSUE THREAD

**Issue by Nicobz22** [BinWang0213/HilbertSort#1]
Dear,

How does the library or module apply to the .msh examples that the file contains?.

the first step is load the module import pyHilbertSort as m, but what is the next step?

Cheers

**Comment by BinWang0213** [comment]
Hi,

Check examples/Gmsh_reordering.ipynb which includes full script to reorder msh file.
https://github.com/BinWang0213/HilbertSort/blob/master/examples/Gmsh_reordering.ipynb

**Comment by BinWang0213** [comment]
@Nicobz22  Does this answer your question? I will close this issue if it is clear.

**Comment by Nicobz22** [comment]
I had thought that the number of vertices, nodes, the program obtained automatically from the Msh file, but I see that you have to manually place them in the program

**Comment by BinWang0213** [comment]
Yes. I think this is more generic and flexible. I will write a small python script that only for Gmsh and support to run it from command line such:
`reorderGmsh.py -file test.msh -sortNode 1 -sortElement 1`

**Comment by Nicobz22** [comment]
I use Gmsh to generate the meshes, where can I find the reorderGmsh.py?.  I think it is not inside the HilbertSort-master package.

Thanks

**Comment by Nicobz22** [comment]
Thank you very much, it worked very well.
